### ISSUE TITLE
Error in whois

### ISSUE THREAD

**Issue by ghost** [supermarsx/whoisdigger#49]
when i select "whois" in bulk settings i get in all data - "Unavailable"

**Comment by eduardomota** [comment]
My best guess is that the whole app needs refactoring, currently is big mess, reason why i haven't worked on it for a while. I'll pick it up soon

**Comment by eduardomota** [comment]
Also, now looking at your message, are you sure you're using the tool correctly?
![image](https://user-images.githubusercontent.com/17675589/160204328-3c8fee16-d242-416f-92fe-4a68d6bd391d.png)
Unavailable just means the domain you're looking at is unavailable, are you sure the domains are false negatives? From my test seems ok, tell me more about your issue

**Comment by bogdan-ivanishchuk** [comment]
Oh yeap, about availability it's work correctly, i tried to check domain with TLD`s...
But bulk whois return "undefined" in "Status","Registrar","Company","Creation Date","Update Date","Expiry Date"
<img width="822" alt="Знімок екрана 2022-03-27 о 6 48 33 пп" src="https://user-images.githubusercontent.com/98334140/160289609-b0f0c692-ba12-4819-ba4e-c294ed075b3c.png">

**Comment by eduardomota** [comment]
Yes, thats due to using DNS sweep and not falling back to whois when unavailable, since you're using the latest version you'll need to change back to simple whois. This is not correctly implemented yet.

go to appsettings.js
![image](https://user-images.githubusercontent.com/17675589/160294886-64c98953-ffa7-44a6-9985-d2656a0fc7c4.png)

change to whois and save
![image](https://user-images.githubusercontent.com/17675589/160294917-f9282c76-e744-4a15-ad3b-d3b8065ae8ca.png)

pure whois is a LOT slower than DNS sweep but you'll be able to get all the info you need.

**Comment by eduardomota** [comment]
There are a bunch of other parameters to fiddle around, also haven't made the app configurable through the UI yet..

On the parameters stuff you can change timeouts, time between requests, randomization upper and lower bounds.

**Comment by bogdan-ivanishchuk** [comment]
Yea i saw this parameter and changed it. But then program return "error" or "unavailable" for all domain and no whois info. Like this:
<img width="361" alt="Знімок екрана 2022-03-28 о 2 54 07 пп" src="https://user-images.githubusercontent.com/98334140/160392800-4d50605a-3b48-4f8c-9ebe-5f82fbe917fa.png">
<img width="657" alt="Знімок екрана 2022-03-28 о 2 54 21 пп" src="https://user-images.githubusercontent.com/98334140/160392818-834ec3fe-174c-4506-93f0-c3de644a08f6.png">

**Comment by svyrydiukk** [comment]
I have the same problem. How can i fix it?

**Comment by supermarsx** [comment]
This bug was caused by messy code, in process of being resolved so closing this.
